src/serviceorchestration/5_0_0/sysd/serviceorchestration_sysd.tex

```diff
@@ -115,64 +115,64 @@ \subsection{Significant Prior Art}
 There are some key differences, even on conceptual level, between the previous versions (Orchestrator 4.6.x) and this version:
 
 \begin{itemize}
-    \item The previous versions were named the system as Orchestrator. Because this expression has a different meaning in some related domains, it is decided that the current version uses the name Service Orchestration to avoid confusion.
-    \item The previous versions contained three (or two in earlier versions) kind of orchestration strategies: a store containing simple, peer-to-peer rules, an other store containing more flexible rules and a dynamic orchestration method which used the live data of the Service Registry to achieve its goal. The current version separates the three strategies into three different Service Orchestration systems (but with the same orchestration service operations), and the Local Cloud's administrator can decide which strategy want to support for their use case.
+    \item The previous versions were named the system as Orchestrator. Because this expression has a different meaning in some related domains, it is decided that the current version uses the name ServiceOrchestration to avoid confusion.
+    \item The previous versions contained three (or two in earlier versions) kind of orchestration strategies: a store containing simple, peer-to-peer rules, an other store containing more flexible rules and a dynamic orchestration method which used the live data of the ServiceRegistry to achieve its goal. The current version separates the three strategies into three different ServiceOrchestration systems (but with the same orchestration service operations), and the Local Cloud's administrator can decide which strategy want to support for their use case.
     \item There was no data storage separation requirement: the Orchestrator's data storage was interconnected to other systems' storage. In the current version, data storage separation is mandatory. 
-    \item You could only create an orchestration rule if all related entities (consumer system, provider system, service definition and interface) are already existed (in the Service Registry's data storage). The current version supports rules referencing non-existent entities for future usage. 
+    \item You could only create an orchestration rule if all related entities (consumer system, provider system, service definition and interface) are already existed (in the ServiceRegistry's data storage). The current version supports rules referencing non-existent entities for future usage. 
```

Comment:
funny, but I think we should use Service Registry here, because we talk about the 4th generation.